dojo/tools/mozilla_observatory/parser.py

```diff
@@ -48,7 +48,7 @@ def get_findings(self, file, test):
                 severity=self.get_severity(int(node['score_modifier'])),
                 static_finding=False,
                 dynamic_finding=True,
-                vuln_id_from_tool=node['name']
```

Comment:
Was the name attribute removed? Might be better to attempt to use the name attribute if available, and then use the key value as a backup. Increased support there.
`node.get('name', key)` should work here

Reply:
Yes, the name is no longer included in the output of the local scan. https://github.com/mozilla/http-observatory/blob/0b7928ee9d09a3f5dffdb81f27d3e5f80ef2024c/httpobs/scanner/local.py#L76

My observation is based on the output of running Mozilla Observatory locally (using the included script). I'm not certain if other options behave the same. However, in all other examples of files that I have seen (also from the unit tests), the name attribute was always identical to the key.

Would you prefer using the key as 'primary source' in all cases or your option as only using it as fallback?

Reply:
Got it. If the name is no longer present, then searching for key as the 'primary source' and then name as the fallback would be fine.

Reply:
I changed it to your initial suggestion `node.get('name', key)` - it should be the most fail safe solution with both formats as it does not change the behavior for files with the name attribute. Although key should (!) be identical to the name attribute also in the older format version and thus safe to use, I cannot be certain (since the only 'old' version files I currently have are the unit test files). A quick test with observatory-cli (https://github.com/mozilla/observatory-cli) showed that the json output here still _does_ contain the name attribute, but also identical to the key.